added extended support section (#4114)
Particular/docs.particular.net: nservicebus/upgrades/support-policy.md

WARNING: If support is required for a version that is not listed or has expired, contact [Particular Software Support](https://particular.net/contactus).

Particular will apply critical bug fixes to all supported versions. If the bug also impacts a non supported version, the recommended approach is to upgrade to a supported version which contains the fix.

For more information refer to [the release policy](/nservicebus/upgrades/release-policy.md).

include: supported-versions-nservicebus

For other NServiceBus packages, refer to [the list of supported versions of all packages](supported-versions.md).

## Extended Support

Extended support is helpful when more time is required to upgrade an existing system from an unsupported version of NServiceBus to a supported version. In order to be entitled to extended support, a valid support agreement with Particular Software is required. Extended support provides an extra two years at the end of the mainstream support. During this period:

- Samples for unsupported versions are still provided for all customers.

> - If a critical bug in an unsupported version is detected and reported, Particular Software will provide a patch release.

Reviewer: After reading this again, I'm not sure it's clear that we will only do this if the bug is reported by a customer which has purchased extended support.

Author: @adamralph Should we also mention that we won't proactively backport the bugs? Can you suggest an edit?

Reviewer: @HEskandari sure, I'll see what I can do.

For other NServiceBus packages, refer to [the list of supported versions of all packages](supported-versions.md).

> ## Extended Support Policy

Reviewer: ```suggestion
## Extended Support
```

This is just a section in _the support policy_.

> Extended support is helpful when more time is needed to upgrade an existing system on an unsupported version of NServiceBus to a supported version of NServiceBus. In order to be entitled to extended support, a valid support agreement with Particular is needed. Extended support provides an extra two years at the end of the mainstream support. During this period:

Reviewer: ```suggestion
Extended support is helpful when more time is required to upgrade an existing system from an unsupported version of NServiceBus to a supported version. In order to be entitled to extended support, a valid support agreement with Particular Software is required. Extended support provides an extra two years at the end of the mainstream support. During this period:
```

>    - Older samples are still provided to customers.

Reviewer: ```suggestion
   - Samples for unsupported versions are still provided for all customers.
```

>    - If a bug is detected and reported, Particular will triage it and provide a patch if it is deemed critical.

Reviewer: ```suggestion
   - If a critical bug in an unsupported version is detected and reported, Particular Software will provide a patch release.
```

> 

Reviewer 2: Are you running your system on an unsupported version of NServiceBus? Extended Support provides you with two more years of support, beginning at the end of mainstream support, allowing you to upgrade your system at your own pace. In order to purchase extended support, a valid support agreement with Particular is needed.  

During this period:
* Older samples continue to be provided to customers
* If a bug is detected and reported, Particular will triage it and provide a patch - as long as it is deemed critical

Reviewer: This suggestion won't work because it uses personal voice (the build will fail).
